models/all/src/migration.ts

```diff
@@ -17,17 +17,11 @@ import { MigrateOperation } from '@anticrm/model'
 
 // Import migrate operations.
 import { coreOperation } from '@anticrm/model-core'
-import { taskOperation } from '@anticrm/model-task'
```

Comment:
I recommend to not remove them, since I assume a lot of times we will need them back.

Reply:
Done